frontend/packages/console-dynamic-plugin-sdk/src/extensions/navigation.ts

```diff
@@ -0,0 +1,69 @@
+import { Extension } from '@console/plugin-sdk/src/typings/base';
+
+export namespace ExtensionProperties {
```

Comment:
Can you add `NavSection` as well. This was a recent addition in order to

`console.navigation/section`
```
  export interface NavSection {
    /** A unique identifier for this item. */
    id: string;
    /** The perspective ID to which this item belongs to. If not specified, contributes to the default perspective. */
    perspective?: string;
    /** Name of this section. If not supplied, only a separator will be shown above the section. */
    name?: string;
    /** Insert this item before the item referenced here. For arrays, the first one found in order is used. */
    insertBefore?: string | string[];
    /** Insert this item after the item referenced here. For arrays, the first one found in order is used. `insertBefore` takes precedence. */
    insertAfter?: string | string[];
  }
```